src/Component/Http/HttpGuzzlClient.php

```diff
@@ -40,16 +47,15 @@ public function __construct(
      */
     public function get($sUrl)
     {
+        $_aRequestOptions = [
+            'connect_timeout' => self::DEFAULT_CONNECTION_TIMEOUT,
+            'timeout' => self::DEFAULT_TIMEOUT
+        ];
+        $_aRequestOptions = $this->addAuthOption($_aRequestOptions);
```

Comment:
I prefer something like "getRequestOptions()" and a method to give you the complete option array instead of duplicate addAuthOption() call - also i would like to see a class cache for the request options that the merge has be execute only one time in a process.